src/pages/Login.jsx

```diff
@@ -0,0 +1,203 @@
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import styles from "./Login.module.css";
+
+const Signup = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [emailError, setEmailError] = useState(null);
+  const [passwordError, setPasswordError] = useState(null);
+  const [isButtonDisabled, setIsButtonDisabled] = useState(true);
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const Navigate = useNavigate();
+
+  // 이메일 값 체크
+  const EmailCheck = (email) => {
```

Comment:
컴포넌트 내부 함수는 소문자로 시작해야해요! (대문자 시작은 컴포넌트 이름만!)
```suggestion
  const emailCheck = (email) => {
```